crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -494,12 +504,18 @@ impl<'a> Submitter<'a> {
                         "submitted transaction",
                     );
                     transactions.push((handle, gas_price));
+                    tx_consecutively_underpriced = 1.;
                 }
                 Err(err) => {
                     tracing::warn!(
                         submitter = %submitter_name, ?err,
                         "submission failed",
                     );
+                    if err.to_string().contains("underpriced") {
+                        tx_consecutively_underpriced += 1.;
```

Comment:
Shouldn't this be:
```suggestion
                        tx_consecutively_underpriced *= GAS_PRICE_BUMP;
```

Reply:
Good catch! My math doesn't add up but I think would still be cleaner to increment `tx_consecutively_underpriced` by 1. and compute the gas price with `GAS_PRICE_BUMP.powi(tx_consecutively_underpriced)`.
No strong feelings, though.

Reply:
Works the same - I had thought about suggesting that as well.